src/main/java/nl/novi/TechItEasy/models/WallBracket.java

```diff
@@ -21,4 +23,11 @@ public class WallBracket {
     private String name;
     private Double price;
 
+    @ManyToMany
+    @JoinTable(
```

Comment:
de `@`JoinTable annotatie is niet verplicht, maar super handig om te weten (te definieren) hoe je de tussentabel moet aanspreken in je data.sql. Handige toevoeging dus